app/controllers/applications_controller.rb

```diff
@@ -7,10 +7,10 @@ def create
     application = Application.new(application_params)
     if application.save
       favorites.pets_applied(application, params)
-      flash.now[:notice] = "Your application for ... has been submitted."
+      flash[:notice] = "Your application for ... has been submitted."
```

Comment:
... is currently a place holder, we'd have to iterate over the params to get the IDs of the pets, and then have a dynamic message